public/app/plugins/panel/annolist/AnnoListPanel.tsx

```diff
@@ -90,7 +90,7 @@ export class AnnoListPanel extends PureComponent<Props, State> {
     const { queryUser, queryTags } = this.state;
 
     const params: any = {
-      tags: options.tags,
+      tags: options.tags.map((e) => getTemplateSrv().replace(e)),
```

Comment:
I think you should use the `replaceVariables` function instead.
```suggestion
      tags: options.tags.map((e) => this.props.replaceVariables(e)),
```